From 4b8d2c8b967087468128caa6ef33719bb965ebff Mon Sep 17 00:00:00 2001
From: Daniel Cordero <theappleman@gmail.com>
Date: Sun, 7 Dec 2008 09:33:30 +0000
Subject: [PATCH] t9120: Prevent test failure for platform error

This test will always fail (and be broken) while SVN_HTTPD_PORT is not
set.
---
 t/t9120-git-svn-clone-with-percent-escapes.sh |   22 +++++++++++-----------
 1 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/t/t9120-git-svn-clone-with-percent-escapes.sh b/t/t9120-git-svn-clone-with-percent-escapes.sh
index ef2c052..1a85491 100755
--- a/t/t9120-git-svn-clone-with-percent-escapes.sh
+++ b/t/t9120-git-svn-clone-with-percent-escapes.sh
@@ -6,26 +6,26 @@
 test_description='git svn clone with percent escapes'
 . ./lib-git-svn.sh
 
-test_expect_success 'setup svnrepo' '
-	mkdir project project/trunk project/branches project/tags &&
-	echo foo > project/trunk/foo &&
-	svn import -m "$test_description" project "$svnrepo/pr ject" &&
-	rm -rf project &&
-	start_httpd
-'
-
 if test "$SVN_HTTPD_PORT" = ""
 then
-	test_expect_failure 'test clone with percent escapes - needs SVN_HTTPD_PORT set' 'false'
+	say 'test clone with percent escapes - needs SVN_HTTPD_PORT set'
 else
+	test_expect_success 'setup svnrepo' '
+		mkdir project project/trunk project/branches project/tags &&
+		echo foo > project/trunk/foo &&
+		svn import -m "$test_description" project "$svnrepo/pr ject" &&
+		rm -rf project &&
+		start_httpd
+	'
+
 	test_expect_success 'test clone with percent escapes' '
 		git svn clone "$svnrepo/pr%20ject" clone &&
 		cd clone &&
 			git rev-parse refs/${remotes_git_svn} &&
 		cd ..
 	'
-fi
 
-stop_httpd
+	stop_httpd
+fi
 
 test_done
-- 
1.6.1.150.g6aac

